src/internal/views/nunjucks/billing/batch-summary.njk

```diff
@@ -2,6 +2,26 @@
 {% from "govuk/components/button/macro.njk" import govukButton %}
 {% from "./nunjucks/billing/macros/totals.njk" import billRunTotal, invoicesTotal, creditNoteTotal %}
 
+{% macro batchLicenceNumbers(licences) %}
+  {% if licences.length > 4 %}
+    <p class="govuk-body govuk-!-margin-bottom-1">{{ licences.length }} licences</p>
+    <details class="govuk-details govuk-!-margin-bottom-0" data-module="govuk-details">
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">View licences</span>
+      </summary>
+      <div class="govuk-details__text">
+        {% for licence in licences %}
+          <div>{{ licence.licenceRef }}</div>
```

Comment:
Do the divs provide sufficient separation of the content for a screenreader?
If not maybe we need an un-styled list <ul> element